Content.Shared/Interfaces/Chemistry/IPlantMetabolizable.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Content.Shared.Chemistry;
+using JetBrains.Annotations;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Serialization;
+
+namespace Content.Shared.Interfaces.Chemistry
+{
+    public interface IPlantMetabolizable : IExposeData
+    {
+        /// <summary>
+        ///     Metabolize 1 unit of a reagent.
+        /// </summary>
+        /// <param name="plantHolder"></param>
+        /// <param name="reactVolume"></param>
```

Comment:
seems like it should actually be `Metabolize <paramref name="reactVolume"/> units of a reagent.`

Reply:
Changed to be proper now